.github/workflows/workflow.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - uses: actions/setup-python@v2
         if: steps.program-changes.outputs.cache-hit != 'true' || steps.conda-dependencies.outputs.cache-hit != 'true'
         with:
-          python-version: 3.x
+          python-version: 3.8.2
```

Comment:
Is this change required? Conda configuration should be taking care of installing the proper Python version.

Reply:
Yes this should be removed. This was introduced during my initial investigation into why the GitHub workflow isnt behaving as it should locally.